### DISCUSSION TITLE
breakage temperature at FDS

### DISCUSSION THREAD

**Discussion by MohamedSalah989** [firemodels/fds#12776] [Q&A] [answered]
Hi FDS team,
How can I identify the breakage temperature for objects like windows in the Pyrosim or FDS models?

**Comment by Er9y714** [comment]
If you mean: triggering an OBSTruction to disappear (analogue to complete fall out glass / breaking) based on a temperature, yes you can do that by adding a conditon to OBSTructions.

**Reply by MohamedSalah989** [reply]
@Er9y714 
 I which field can i identify that? 
 
![Obstruction](https://github.com/firemodels/fds/assets/138606386/2e2924bd-333c-44bd-a149-e5491922527b)

**Reply by Er9y714** [reply]
I recommend to review the documents @drjfloyd mentioned. As far as I know, Pyrosim (2023.3.1206) does not offer an option to link a device (other than Time) with obstructions. You can either create a CTRL (control) or add DEVC_ID of temperature or any other device to the obstruction line (Advanced tab). You will see these practices (FDS inputs) while reading the referred documents in @drjfloyd 's post.

**Reply by MohamedSalah989** [reply]
@Er9y714 
Many thanks for your answer; I will follow this. It is highly appreciated!.

**Comment by drjfloyd** [comment]
First define what the specific criterion is that you wish to use to determine when a window should break.  For example: "A window should break out completely when the maximum radiative flux to the window is 50 kW/m2"

Then read the User Guide chapter on FDS outputs and think about what outputs or combinations of outputs would allow FDS to determine when  the window should break. For the example above you might think to look for how do I measure radiative flux and how can I get the maximum value over an area.

Then read the User Guide chapter on control logic to learn how you can remove the window based on some measurment you have made with FDS outputs.

**Reply by MohamedSalah989** [reply]
@drjfloyd 
Many thanks for your answer; I will follow this. It is highly appreciated!.

**Comment by MohamedSalah989** [comment]
Dear @drjfloyd 
Do you have a FDS sample for such a case that i can follow as a reference for my setup. I will be grateful for your support.

**Reply by Er9y714** [reply, marked as answer]
See User Guide 18.4.1 Creating and Removing Obstructions

**Reply by MohamedSalah989** [reply]
@Er9y714 
Many thanks, for your great guidance; I solved the issue.
